.travis.yml

```diff
@@ -83,5 +83,10 @@ matrix:
 # Use travis_retry during install to reduce the impact of network failures.
 before_install: travis_retry ./Support/Testing/Travis/before-install.py
 install:        travis_retry ./Support/Testing/Travis/install.py
+before_script:
```

Comment:
Should we put that in a separate file ? it sound like it will be run for all target too